treeherder/model/migrations/0035_bugscache_optional_bugzilla_ref.py

```diff
@@ -76,4 +76,11 @@ class Migration(migrations.Migration):
                 ),
             ],
         ),
+        migrations.AddConstraint(
+            model_name="BugscacheOccurrence",
+            constraint=models.UniqueConstraint(
+                fields=("user", "failure_line", "bug"),
```

Comment:
The condition could also be `fields=("failure_line", "bug")` - there is no value in having the same classification multiple times by different people.